app/test_main.py

```diff
@@ -2,0 +1,15 @@
+from unittest.mock import patch
+from typing import Callable
+
+from app.main import can_access_google_page
+
+
+@patch("app.main.valid_google_url", reutn_value=True)
+@patch("app.main.has_internet_connection", reutn_value=True)
```

Comment:
There is a typo in the `@patch` decorator. It should be `return_value` instead of `reutn_value`.